User-Help/Articles.md

```diff
@@ -0,0 +1,9 @@
+## Articles and blogs
```

Comment:
I've seen some confusion out there already that conflates categories and post types, so I'd like our doc to be clear on that.  Categories and post types are orthogonal.  A category can have more than one type of top-level post (currently, that means a category can have both Q&A and articles).  On Judaism somebody proposed that for Purim Torah, so there could be funny questions and funny articles all together in the PT category.  (That might not happen so don't enshrine it in help; that's just an example for you.)

Articles are a way to *implement* blogs, resources, wikis, etc, but those aren't the only uses for articles.

In this topic, could we start by describing what articles are (how they differ from questions), and then talk about ways they can be used?

Reply:
Done. I ended up basically rewriting it to explain both different post types as well as categories.

Reply:
Much better!  Sorry to prompt a whole rewrite there... This looks good now.  Where you're giving examples of categories, consider adding the Meta blog -- doesn't hurt to get it a little more visibility throughout the network, since many people will probably come for a specific community and not look at Meta otherwise.  (This is in addition; don't remove the examples from the other sites.)